src/libpmem2/x86_64/memcpy/memcpy_t_movdir64b.c

```diff
@@ -0,0 +1,440 @@
+// SPDX-License-Identifier: BSD-3-Clause
```

Comment:
movdir64b are always non-temporal. This (and memset) doesn't make much sense. It would be better to just fallback (in the higher layer) to the next best temporal version.